src/foraging_gui/Foraging.py

```diff
@@ -2140,11 +2432,15 @@ def _Save(self,ForceSave=0,SaveAs=0,SaveContinue=0):
         if hasattr(self, 'LaserCalibrationResults'):
             self._GetLaserCalibration()
             Obj['LaserCalibrationResults']=self.LaserCalibrationResults
+        else:
```

Comment:
Most sessions wont have a laser calibration result, why include an empty json?

Reply:
This was to ensure that the two checkpoint and end-of-session files always matched. Can be removed.